src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs

```diff
@@ -79,7 +79,10 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
         => compilation.GetTypeByMetadataName(typeof(System.Diagnostics.DebuggerDisplayAttribute).FullName!);
 
     public static INamedTypeSymbol? StructLayoutAttributeType(this Compilation compilation)
-        => compilation.GetTypeByMetadataName(typeof(System.Runtime.InteropServices.StructLayoutAttribute).FullName!);
+        => compilation.GetTypeByMetadataName(typeof(StructLayoutAttribute).FullName!);
+
+    public static INamedTypeSymbol? InlineArrayAttributeType(this Compilation compilation)
+        => compilation.GetTypeByMetadataName("System.Runtime.CompilerServices.InlineArrayAttribute");
```

Comment:
>InlineArrayAttribute [](http://example.com/codeflow?start=78&length=20)

can typeof not be used here too?

Reply:
no.  this compiles against sdks that doesn't have this type itself.